src/ActionList/Item.tsx

```diff
@@ -254,7 +260,11 @@ const TrailingVisualContainer = styled(ColoredVisualContainer)`
 
 const DescriptionContainer = styled.span<{descriptionVariant: ItemProps['descriptionVariant']}>`
   color: ${get('colors.text.secondary')};
+  font-size: ${get('fontSizes.0')};
```

Comment:
Per https://github.com/github/primer/issues/41, 12px fonts should have a 16px line-height:

<img width="821" alt="Part of the Typographic Scale defined in github/primer#41" src="https://user-images.githubusercontent.com/3104489/119725556-1553df00-be3e-11eb-902f-47f485abd67e.png">


| 12px/16px (spec) | 12px/20px (this PR) |
|:---:|:---:|
| <img width="311" alt="A description rendered with a 12px font-size and a 16px line-height" src="https://user-images.githubusercontent.com/3104489/119725443-ee95a880-be3d-11eb-8468-2ef8a82c77f8.png"> | <img width="312" alt="A description rendered with a 12px font-size and a 20px line-height" src="https://user-images.githubusercontent.com/3104489/119725495-03723c00-be3e-11eb-9ccb-82155be68c83.png"> |

Reply:
Great catch! I updated it to `16px` and also added an `align-items: baseline` in the text section so that the main text and description line up as they do in the designs
![image](https://user-images.githubusercontent.com/3026298/119763975-688d5800-be65-11eb-97ed-4c36c18a6ac7.png)